content/applications/finance/accounting/taxes/avatax.rst

```diff
@@ -1,74 +1,355 @@
    :align: center
    :alt: Configure Avatax settings
 
-.. _avatax/tax-mapping:
+Prerequisites
+-------------
 
-Tax mapping
-===========
+Before the credentials are entered in, tick the checkbox to the right of the :guilabel:`AvaTax`
+title to activate the feature.
 
-The Avatax integration is available on Sale Orders and Invoices with the included Avatax fiscal
-position.
+Next, select the :guilabel:`Environment` that the company wishes to use AvaTax in. It can either be
+:guilabel:`Sandbox` or :guilabel:`Production`.
 
-Before using the integration, specify an :guilabel:`Avatax Category` on the product categories.
+.. seealso::
+   For help determining which *AvaTax* environment to use (either :guilabel:`Production` or
+   :guilabel:`Sandbox`), visit: `Sandbox vs Production environments
+   <https://knowledge.avalara.com/bundle/fzc1692293626742/page/sandbox-vs-production.html>`_.
 
-.. image:: avatax/avatax-category.png
-   :align: center
-   :alt: Specify Avatax Category on products
+Credentials
+-----------
 
-Avatax Categories may be overridden or set on individual products as well.
+Now, the credentials can be entered in. The AvaTax :guilabel:`Account ID` will be entered in the
+:guilabel:`API ID` field and the :guilabel:`License Key` will be entered in the :guilabel:`API Key`
+field.
 
-.. image:: avatax/override-avatax-product-category.png
-   :align: center
-   :alt: Override product categories as needed
+.. tip::
+   The :guilabel:`Account ID` can be found by logging into the AvaTax portal (`sandbox
+   <https://sandbox.admin.avalara.com/>`_ or `production <https://admin.avalara.com/>`_). In the
+   upper-right corner are the initials of the user and :guilabel:`Account`, click into this icon.
+   The :guilabel:`Account ID` is listed first.
+
+   To access the :guilabel:`License Key` see this documentation:
+   :ref:`avatax/create_avalara_credentials`.
+
+Transaction options
+-------------------
+
+There are two transactional settings that can be configured: :guilabel:`Use UPC Commit` and
+:guilabel:`Commit Transactions`.
 
-.. _avatax/address-mapping:
+If the checkbox next to :guilabel:`Use UPC codes` is ticked then, the transactions will use
+Universal Product Codes (UPC), instead of custom defined codes in Avalara. Consult a certified
+public accountant (CPA) for specific guidance.
+
+Should the :guilabel:`Commit Transactions` checkbox be ticked, then, the transactions in the Odoo
+database will be committed for reporting in AvaTax.
 
 Address validation
-==================
+------------------
 
-Manually validate customer addresses by clicking the :guilabel:`Validate address` link in the
-customer form view.
+.. important::
+   The :guilabel:`Address validation` feature only works with partners in North America.
 
-.. image:: avatax/validate-customer-address.png
-   :align: center
-   :alt: Validate customer addresses
+Additionally, tick the checkbox next to the :guilabel:`Address validation` field.
```

Comment:
You did a wonderful job explaining the previous settings. It would be really helpful if you could do the same here. In a nutshell, could you tell me what the Address Validation field does? Thanks!

```suggestion
Additionally, tick the checkbox next to the :guilabel:`Address Validation` field.
```